desktop/main.js

```diff
@@ -80,6 +81,51 @@ const quitAndInstallWithUpdate = () => {
     autoUpdater.quitAndInstall();
 };
 
+/**
+ * Menu Item callback to triggers an update check
+ * @param {MenuItem} menuItem
+ * @param {BrowserWindow} browserWindow
+ */
+const manuallyCheckForUpdates = (menuItem, browserWindow) => {
+    // Disable item until the check (and download) is complete
+    // eslint: menu item flags like enabled or visible can be dynamically toggled by mutating the object
+    // eslint-disable-next-line no-param-reassign
+    menuItem.enabled = false;
```

Comment:
Can we just create a local variable here? Instead, adding
// eslint-disable-next-line no-param-reassign

Reply:
We set the `menuItem.enabled` to `false` to display the item disabled in the menu (while the check is running or while update is being downloaded) 

Having a local variable would not make the `menuItem` appear as disabled 

We can change this to 

```jsx
// eslint-disable-next-line no-use-before-defined
checkForUpdateMenuItem.enabled = false;
```

It still requires an `eslint` override, because the actual menu item is declared on line 149

Reply:
To toggle a menu item, you can mutate the `menuItem` object or rebuild the system menu